src/context.cc

```diff
@@ -22,6 +22,8 @@ SimInfo::SimInfo()
       m0(0),
       dt(kDefaultTimeStepDur),
       decay("manual"),
+      seed(kDefaultSeed),
+      stride(kDefaultStride),
```

Comment:
These should be initialized in the order they are declared in the header file.  You can either change all of these, or move the location in which they are ordered in the header file.  Note: this is a best practice and is enforced with some compiler options.  It may not be true of all current Cyclus practice.